features/fixtures/statistics.json

```diff
@@ -25,7 +25,6 @@
   "details":{
     "document_noun":"statistic",
     "filter":{
-      "content_purpose_supergroup": "research_and_statistics"
```

Comment:
Does the stats supergroup finder no longer need this field?

Reply:
The test doesn't really - this was part of testing cps->content types conversion